grpc/proto/space.proto

```diff
@@ -365,6 +365,7 @@ message SharedListDirectoryEntry {
   ListDirectoryEntry entry = 1;
   string dbId = 2;
   string bucket = 3;
+  bool isPublicLink = 4;
```

Comment:
Don't we also have to add the `password` field to the proto?

Reply:
No, it is not useful. After the first time the open public file rpc is called, because that logic persists the password and automatically refetches it on subsequent calls.

So the frontend would pass an empty password when opening public link files from the shard with me response

Reply:
How do they pass the password then?

Reply:
They pass it only on the `OpenPublicFile` rpc request

Reply:
I don't see a field for password there

Reply:
I don't see a field for password there